Doc/library/functions.rst

```diff
@@ -196,7 +196,7 @@ are always available.  They are listed here in alphabetical order.
    base 16).  :exc:`ValueError` will be raised if *i* is outside that range.
 
 
-.. function:: classmethod(function)
+.. decorator:: classmethod(function)
```

Comment:
I suspect that this should be `decorator: classmethod`; the arguments syntax is probably used for functions that are decorator factories.

(For example, `@functools.lru_cache(max_size=20)` gets arguments and returns a decorator which is then applied to the decorated function; the markup must show clearly if people are expected to call `@decorator` or `@decorator_factory()`.)